swift-generator/pom.xml

```diff
@@ -93,5 +93,31 @@
       <groupId>io.airlift</groupId>
       <artifactId>log</artifactId>
     </dependency>
+    
+    <dependency>
+      <groupId>com.facebook.swift</groupId>
+      <artifactId>swift-test-utils</artifactId>
+      <scope>test</scope>
+    </dependency>
+    
+    <dependency>
+      <groupId>org.testng</groupId>
+      <artifactId>testng</artifactId>
+      <scope>test</scope>
+    </dependency>
+    
   </dependencies>
+  
+  <build>
+      <plugins>
+          <plugin>
```

Comment:
This is needed to set the test generated output root directory - that is useful to see the output after each Thrift file after the run - another reason why it isn't a temp directory.